src/server/game/World/World.h

```diff
@@ -155,6 +157,8 @@ class World: public IWorld
     World();
     ~World() override;
 
+    std::list<DelayedDamage> _delayedDamages;
```

Comment:
can make private and why `std::list` ?

Reply:
i needed something that supports constant time insertion and removal of elements from anywhere in the container. do you have any recommendations for something else?

Reply:
testing with it being moved private. at the moment moving it to private seems to regress the PR to where the animation is hardly playing in full at all.